packages/flutter_tools/templates/app/android.tmpl/app/src/main/res/values-night/styles.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Theme applied to the Android Window while the process is starting -->
```

Comment:
Add more context to this comment. e.g. `Theme applied to the Android Window while the process is starting when the OS's Dark Mode setting is on` in this and the next file.